src/app/modals/user/registered-form.view.ts

```diff
@@ -1,21 +1,26 @@
 import { VirtualDOM } from '@youwol/flux-view'
 import { Accounts } from '@youwol/http-clients'
-import { BaseUserFormView, separatorView, redirectWith } from './common'
+import { separatorView, redirectWith } from './common'
+import { ProfilesBadgeView } from '../../top-banner/badges'
 
 /**
  * @category View
  */
-export class RegisteredFormView extends BaseUserFormView {
```

Comment:
Is BaseUserFormView still used, if not it should be deleted (not sure it has been deleted)

Reply:
It is done now, check it.